common/compositor/compositorthread.h

```diff
@@ -30,6 +30,8 @@
 #include "fdhandler.h"
 #include "hwcevent.h"
 
+#include "framebuffermanager.h"
```

Comment:
instead of having header here, you can just have a forward declaration i.e. class FrameBufferManager: